crates/proof-of-sql/src/base/database/column.rs

```diff
@@ -266,6 +303,9 @@ pub enum ColumnType {
     /// Mapped to [`Curve25519Scalar`](crate::base::scalar::Curve25519Scalar)
     #[serde(alias = "SCALAR", alias = "scalar")]
     Scalar,
+    /// Mapped to fixed size binary
+    #[serde(alias = "FIXEDSIZEBINARY", alias = "fixedsizebinary")]
+    FixedSizeBinary(i32),
```

Comment:
At first, I wanted to use `FixedSizeBinary(usize)` since it made more sense to me, but [Arrow actually uses `i32` for that](https://docs.rs/arrow/latest/arrow/datatypes/enum.DataType.html#variant.FixedSizeBinary), so I decided to stick with the convention.

> having some way of ensuring that the number of bytes must be in the range `[0, 32]`


where do you think such a check should go?